testing/test_api_credentials.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+import pytest
+import mock
+
+from manageiq_client.api import ManageIQClient as MiqApi
+
+
+def test_no_user_password_dict_list_or_tuple():
+    with pytest.raises(ValueError):
+        MiqApi('http://www.test.com', {})
+        MiqApi('http://www.test.com', [])
+        MiqApi('http://www.test.com', ())
+
+
+def test_no_password_dict():
+    with pytest.raises(KeyError):
```

Comment:
consider making this too a ValueError, for consistency (`if "user" in auth and "password" in auth`)
LGTM either way.